mlir/include/mlir/Dialect/Arith/IR/ArithBase.td

```diff
@@ -156,4 +156,29 @@ def Arith_IntegerOverflowAttr :
   let assemblyFormat = "`<` $value `>`";
 }
 
+//===----------------------------------------------------------------------===//
+// Arith_RoundingMode
+//===----------------------------------------------------------------------===//
+
+// These correspond to LLVM's values defined in:
+// llvm/include/llvm/ADT/FloatingPointMode.h
```

Comment:
Given that the arith dialect operations could be converted to the APIRV dialect these enums need not match the LLVM values.

Reply:
They don't exactly match (`tonearesteven` vs `tonearest`)... Maybe we can just drop that claim.